edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
```

Comment:
hrm. I hope we didn't overlook anything. Any change you happen to remember which PR?